_episodes/16-parallel.md

```diff
@@ -125,7 +575,7 @@ Create a submission file, requesting more than one task on a single node:
 {{ site.sched.comment }} {{ site.sched.flag.queue }} {{ site.sched.queue.testing }}
 {% include {{ site.snippets }}/parallel/four-tasks.snip %}
 module load python3
-mpirun ./pi.py 1431652028
+mpirun python pi.py 100000000
```

Comment:
This also needs to be configurable:
```
{{ site.remote.mpiexec }} {{ site.remote.mpiexec_args }} {{ site.remote.python }}  pi.py 100000000
```
but this already starts to look a bit tedious, perhaps the jobscript is better off in a snippet

Reply:
I've replaced all the `mpirun` commands with `mpiexec` -- I'm good to make it configurable for `srun` or other commands, whether in a snippet or a variable. Could that be a future issue and PR if the remainder of the episode is acceptable?

Reply:
I'm good for either option, as mentioned elsewhere, but if we can get this merged in and decide on that in a separate issue/PR, that'd help.